Runtime/Interaction/SnapZone.cs

```diff
@@ -251,10 +321,9 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
         /// </summary>
         protected virtual void DrawHighlightMesh()
         {
-            if (previewMesh != null)
+            if (PreviewMesh != null)
             {
-                Matrix4x4 matrix = Matrix4x4.TRS(attachTransform.position, transform.rotation, transform.localScale);
-                Graphics.DrawMesh(previewMesh, matrix, previewMaterial, gameObject.layer);
+                Graphics.DrawMesh(PreviewMesh, attachTransform.localToWorldMatrix, activeMaterial, gameObject.layer);
```

Comment:
Please consider sub meshes, see: https://github.com/Innoactive/XR-Interaction-Component/pull/46

Reply:
not needed as the preview model is explicitly prepared (merges submeshes)

Reply:
Only if it was created from the snap zone generator, if the user manually creates a snap zone and uses a custom highlight model with submeshes, we will have a bug report.

Also, the implementation is completely safe if the model has no submeshes.